Fixes biomass packets.

Eneocho opened the pull request:
The packet_override var on the packet was valid for powercreep but runtimed on biomass, which broke the biomass packet.
### What this does
Fixes biomass packets by removing the packet_override var.
Fixes #32958
[hotfix][tested]
:cl:
 * bugfix: Fixes biomass packets not working. As a side effect of this, powercreep spawning from a packet has a slightly shorter animation on creation.

gurfan commented:
> The packet_override var on the packet was valid for powercreep but runtimed on biomass, which broke the biomass packet.
> 
> ### What this does
> Fixes biomass packets by removing the packet_override var. Fixes #32958 [hotfix] 🆑
> 
> * bugfix: Fixes biomass packets not working.

hey just wondering did you ever test the thing you added at all

Eneocho commented:
I did test it when making it, but without the packet_override var. When kurf asked me to rework the packets to use inheritance I added the packet override back, which broke the biomass packets.
This is tested btw. I removed the packet override from the deployable packet and powercreep. The only difference between powercreep before this and after this is a shorter animation upon spawning from the packet.
